Source/NSWindow.m

```diff
@@ -5022,9 +5024,6 @@ - (void) setFrameFromString: (NSString*)string
         }
     }
 
-  // Make sure we are using the new screen we are applying to...
```

Comment:
You probably removed this, because the later interaction will also try to set the screen. But now we have a window that does not have a screen assigned for some time. Is this a problem?

Reply:
This is not a problem because window has screen assigned in initializer methods (initWithContentRect...).